### ISSUE TITLE
exit install on dependencies unmet (for source install)

### ISSUE THREAD

**Issue by MisterRayCo** [DataDog/dd-agent#538]
Should stop or try another when getting:
Cannot fetch index base URL http://d.pypi.python.org/simple/

Setting up supervisor.....Downloading/unpacking supervisor
Cannot fetch index base URL http://d.pypi.python.org/simple/
Could not find any downloads that satisfy the requirement supervisor
No distributions at all found for supervisor
Storing complete log in /Users/tbd/.pip/pip.log

**Comment by remh** [comment]
This will be fixed by https://github.com/DataDog/dd-agent/pull/1504

**Comment by theckman** [comment]
It looks like #1504 was split in to two subsequent PRs: #2198 and #2199. #2198 has been merged, and #2199 will be merged after the `5.7.0` git tag is cut.

Should this issue be closed, or removed from the `5.7.0` milestone, because #2199 will be merged after `5.7.0` is done?

**Comment by olivielpeau** [comment]
Hi @theckman!

#2199 will be merged after the `5.7.0` release is tagged indeed (for reasons that aren't directly related to the contents of the `5.7.0` release but only to the way the source install script works).

Are you looking for a way to install the current state of the agent on the master branch with the source install script? If so, you can run:

```
AGENT_VERSION=master DD_API_KEY=<your_api_key> sh -c "$(curl -L https://raw.githubusercontent.com/DataDog/dd-agent/olivielpeau/source-install-revamp-script/packaging/datadog-agent/source/setup_agent.sh)"
```

to pull the source install script from #2199 and use it to install the agent from the `master` branch.

Please note that installing the master branch is _not_ officially supported, but if you have feedback on it we'd be glad to hear it.

Let me know if this doesn't address your question/concern.

Thanks!

**Comment by theckman** [comment]
I'm sorry for the ambiguity of my question. I was asking if this issue, #538, should be removed from the `5.7.0` GitHub milestone. If following the milestone, this would need to be closed before `5.7.0` release. But it probably won't be closed until #2199 is closed, which will happen after `5.7.0` release.

**Comment by olivielpeau** [comment]
Thanks for clarifying!

Actually we can be flexible as to when to close issues. This issue (#538) should probably be closed as the PR that fixes it will be shipped with 5.7.0 (even though the commits of that PR will not _technically_ be part of the tag, and even though the PR is not merged yet, for all practical purposes the issue will be fixed with 5.7.0).

Hope that makes sense. I'm going to close this issue, but feel free to add a comment.
